src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -94,17 +96,18 @@ public ProcessResult process(CcdCallbackRequest request, String idamToken, Strin
 
         if (!result.getWarnings().isEmpty()) {
             log.warn(
-                "Warnings found for {} during callback process:\n  - {}",
+                "Warnings found for {} during callback process: {}",
```

Comment:
can we include exception record id here ?

Reply:
not yet possible - needs further refactor/extraction as ID comes from validation (gets included in exception record model)

Reply:
applied